pkg/resources/external_volume_acceptance_test.go

```diff
@@ -0,0 +1,1854 @@
+package resources_test
+
+// TODO Add test that includes Iceberg table creation, as this impacts the describe output (updates ACTIVE)
```

Comment:
Would be good to have a test here that creates an Iceberg table, as this impacts the output of the `ACTIVE` property in the External Volume describe output. I would probably need your help to write this as Snowflake verifies access to the data on creation, so random values can’t be used for the bucket etc. - https://docs.snowflake.com/en/user-guide/tables-iceberg-storage#verifying-storage-access.